test/db/cmd/cmd_afb

```diff
@@ -8,7 +8,7 @@ afb.
 EOF
 EXPECT=<<EOF
 33
-main
+entry0
```

Comment:
looks like a regression

Reply:
Nope. That's what I thought too first, but this is actually more correct. `aa` has this special logic where it forces the name of the function at `entry0` to be called "entry0":
https://github.com/radareorg/radare2/blob/4afa4296ea32a314f1aad22b35926fb892223bf9/libr/core/canal.c#L4072-L4075
So this is the behavior that you would expect and also the one that is there after this pr:
![Screenshot from 2020-04-17 17-22-53](https://user-images.githubusercontent.com/1460997/79585769-90da4d00-80d0-11ea-8ff5-2f938c4522a2.png)
But because before, aanr would re-analyze without this special entry0 treatment, it would call it differently:
![Screenshot from 2020-04-17 17-23-11](https://user-images.githubusercontent.com/1460997/79585854-acddee80-80d0-11ea-9ac6-35438a492ac8.png)
This obviously makes no sense since noreturn propagation shouldn't just randomly change function names.

About this special treatment of entry0, I personally don't like it and I would remove it. But this should be done in another pr.

Reply:
nice